src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -68,6 +69,48 @@ public function updateEntry(Entry $entry, $url, array $content = [], $disableCon
         $this->stockEntry($entry, $content);
     }
 
+    /**
+     * Try to sanitize the title of the fetched content from wrong character encodings and invalid UTF-8 character.
+     * @param $title
+     * @param $contentType
+     * @return string
+     */
+    private function sanitizeContentTitle($title, $contentType) {
+        if ('application/pdf' === $contentType) {
+            $convertedTitle = $this->convertPdfEncodingToUTF8($title);
```

Comment:
You should instead override the `$title` value and use only one `return $this->sanitizeUTF8Text($title);`:

```php
if ('application/pdf' === $contentType) {
    $title = $this->convertPdfEncodingToUTF8($title);
}

return $this->sanitizeUTF8Text($title);
```